scripts/build/Dockerfile.hotspot-alpine

```diff
@@ -1,29 +1,11 @@
 FROM docker.io/library/eclipse-temurin:11-alpine
 ARG CC=gcc
 
-RUN apk update && apk add \
-	bash \
-	build-base \
-	coreutils \
-	git \
-	gnutls-dev \
-	libaio-dev \
-	libcap-dev \
-	libnet-dev \
-	libnl3-dev \
-	pkgconfig \
-	protobuf-c-dev \
-	protobuf-dev \
-	python3 \
-	sudo \
-	maven \
-	ip6tables \
-	iptables \
-	util-linux-dev \
-	bash
+RUN apk update
```

Comment:
Every `RUN` command creates a new layer in the container image. It might be better to use `RUN apk update && apk add $(sed 's/\#.*$//' contrib/apk-packages.txt)`